site/content/docs/02-template-syntax.md

```diff
@@ -1635,11 +1635,11 @@ If `this` is falsy, no component is rendered.
 
 ---
 
-The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when rich text content from a CMS. If the tag is changed, the children will be preserved unless there's a transition attached to the element. Any properties and event listeners present will be applied to the element.
+The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when rich text content from a CMS. Any properties and event listeners present will be applied to the element.
 
 The only supported binding is `bind:this`, since the element type specific bindings that Svelte does at build time (e.g. `bind:value` for input elements) does not work with a dynamic tag type.
 
-If `this` has a nullish value, a warning will be logged in development mode.
+If `this` has a nullish value, element and its children are not displayed.
```

Comment:
```suggestion
If `this` has a nullish value, the element and its children will not be rendered.
```

Reply:
Thank you! I should go to English school! 😅